pkg/plugins/manager/loader/assetpath/assetpath_test.go

```diff
@@ -190,6 +190,21 @@ func TestService(t *testing.T) {
 				require.NoError(t, err)
 				require.Equal(t, oneCDNRelativeURL, u)
 			})
+
+			t.Run("GetTranslations", func(t *testing.T) {
+				pluginInfo := NewPluginInfo(jsonData["one"], plugins.ClassExternal, pluginFS("one"), nil)
+				pluginInfo.pluginJSON.Languages = []string{"en-US", "pt-BR"}
+				translations, err := svc.GetTranslations(pluginInfo)
+				require.NoError(t, err)
+				oneCDNURL, err := url.JoinPath(tc.cdnBaseURL, "/one/1.0.0/public/plugins/one")
```

Comment:
Nit: It's good practice to avoid system specific file path separators. Therefore this should become something like:
```suggestion
				oneCDNURL, err := url.JoinPath(tc.cdnBaseURL, "one", "1.0.0", "public", "plugins", "one")
```

Reply:
Whoops - still too sleepy. This is obviously URL and not file paths 🤦

Reply:
Awesome 🙌 , I'll fix 👍

Reply:
Fixed 👍

Reply:
You probably missed my [subsequent comment](https://github.com/grafana/grafana/pull/102853#discussion_r2015972466) 😅 You didn't need to change this actually - my apologies 😓 But it will still work either way!